### ISSUE TITLE
`install_tool_dependencies: False` is not working

### ISSUE THREAD

**Issue by bgruening** [galaxyproject/ansible-galaxy-tools#29]
```
  install_tool_dependencies: False
  install_repository_dependencies: False
```

Is not working reliable. I get repositories added to the database but the installation is not triggered. This option get more and more important as we try to get first flavours up and running with pure conda/docker dependencies.

**Comment by afgane** [comment]
These config values just get passed on to Galaxy: https://github.com/galaxyproject/ansible-galaxy-tools/blob/master/files/install_tool_shed_tools.py#L580. When you say "not reliable", does this mean running the script multiple times for the same Galaxy or just independent runs on different Galaxy instances?

**Comment by bgruening** [comment]
@afgane it actually never worked for me. I see the repositories in the Galaxy admin menue but they are just white and I need to trigger every installation manually.

**Comment by bgruening** [comment]
I will try to look at this at some point, just wanted to note it somewhere.

**Comment by mvdbeek** [comment]
Following up with this; this happens to repositories with "complex repository dependencies" which also happens when you trigger the installation through the UI. I'd say this is probably something that should be fixed in galaxy, but I'm not sure that's easy.

**Comment by afgane** [comment]
Do we have any updates on this?

**Comment by mvdbeek** [comment]
Nope, not really. https://github.com/galaxyproject/galaxy/issues/3745

**Comment by mvdbeek** [comment]
So, ahm, maybe we should close the issue. 
```
  install_tool_dependencies: False
  install_repository_dependencies: False
```
has always been working fine, if you define` fine` as `it does what galaxy is doing`.

Problem is still the repository type dependencies, like suites and packages that get cloned and enter the database in status `NEW`. We're tracking that in https://github.com/galaxyproject/galaxy/issues/3745
